examples/deployment-manager/autogen/singlevm/mpdev.yaml

```diff
@@ -0,0 +1,118 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: dev.marketplace.cloud.google.com/v1alpha1
+kind: DeploymentManagerTemplate
+metadata:
+  name: dmtemplate
+deploymentManagerRef:
+  group: dev.marketplace.cloud.google.com
+  kind: DeploymentManagerAutogenTemplate
+  name: autogen
+zipFilePath: template.zip
+---
+apiVersion: dev.marketplace.cloud.google.com/v1alpha1
+kind: DeploymentManagerAutogenTemplate
+metadata:
+  name: autogen
+spec:
+  packageInfo:
```

Comment:
This is very clean! I think some comments or links to documentation (which perhaps need to be generated by godoc?) will be helpful.